zeroex/order_test.go

```diff
@@ -111,5 +111,7 @@ func TestMarshalUnmarshalOrderEvent(t *testing.T) {
 	signedOrder.ResetHash()
 
 	require.NoError(t, json.NewDecoder(buf).Decode(&decoded))
-	assert.Equal(t, orderEvent, decoded)
+	// TODO(jalextowle): An `assert` here will allow wasm browser tests to fail regardless
+	// of their actual result. Change this to use `assert` once this has been fixed in `testify`
+	require.Equal(t, orderEvent, decoded)
```

Comment:
I don't think this has anything to do with `testify` and rather is a problem with either our tests or `wasmbrowsertest`. We should be able to go back to using `assert` here after I added the workaround in https://github.com/0xProject/wasmbrowsertest/pull/1.